### PR TITLE
Misc Fixes

### PR DESCRIPTION
**Pull request by ghost**
- makes the archive button clickable
- make the mobile hamburger button visible
- add team section to nav
- fix SVGs on chrome

### CODE REVIEW

File: src/images/svg-2022/space-jam-title.svg [hunk 1]
```diff
@@ -1,82 +1,24 @@
...
-            />
-            <feBlend
-                    mode='normal'
-                    in='SourceGraphic'
-                    in2='effect1_dropShadow_3:3788'
-                    result='shape'
-            />
-        </filter>
-    </defs>
+<svg width="882" height="257" viewBox="0 0 882 257" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jessicasu77) [inline src/images/svg-2022/space-jam-title.svg:1]: Was wondering what was changed about these new SVGs?

**Review by jessicasu77 — APPROVED** [review]
Looks great Alex!!

**Review by jzhzhang66 — APPROVED** [review]
LGTM thanks alex:)

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
WAIT JK the medium icon is gone
Screen Shot 2021-12-28 at 1 23 22 PM

**Comment by ghost** [comment]
Aight lmao im adding percy when i get home

**Review by jzhzhang66 — APPROVED** [review]
Yay medium icon is back!!

**Pull request merged**